src/foam/nanos/dig/drivers/DigFormatDriver.js

```diff
@@ -139,7 +150,7 @@ foam.CLASS({
 
       ClassInfo cInfo = dao.getOf();
       Predicate pred = new WebAgentQueryParser(cInfo).parse(x, q);
-      ((Logger) x.get("logger")).debug("predicate", pred.getClass(), pred.toString());
+      getLogger().debug(pred.toString());
```

Comment:
Is the .toString() required?

Reply:
Probably not .. i'll see about removing it.